### DISCUSSION TITLE
Tracking both internal & outbound clicks

### DISCUSSION THREAD

**Discussion by janoschherrmann** [plausible/analytics#1719] [Self-Hosted Support] [answered]
I'm going through the Plausible documentation right now, and I'm a bit confused. The post on tracking outbound links tells me to change the tracking script, but wouldn't that mean the internal tracking wouldn't work anymore? 

I'd like to track both internal clicks and external link clicks through the global script. I'm using the self-hosted plan, by the way.

**Comment by Vigasaurus** [comment, marked as answer]
All of the script variants (except for manual) work as additions or like extra plugins - i.e. they add functionality only, while the core functions continue to work the same. Manual mode is the only one which turns off automatic pageview tracking.

**Reply by janoschherrmann** [reply]
Ok so when I want to use the normal script and track external links at the same time, I just include both scripts, right?

**Reply by Vigasaurus** [reply]
No - I think calling it a plugin was a bit misleading - just loading the `plausible.outbound-links.js` will track both the default pageviews from the regular script, and will listen for outbound links. You don't need to include the script several times for each extra function. 

Similarly, for example, loading just the `plausible.outbound-links.exclusions.js` will do all of the functionality from the normal script, the outbound link tracking, and the ability for you to specify URL exclusions. (Also, the script extras order doesn't matter in the path either - i.e. you could use `plausible.exclusions.outbound-links.js` as well, for example).

**Reply by janoschherrmann** [reply]
Ah, perfect. Thank you for your help!
